src/CachePlugin.php

```diff
@@ -225,6 +224,10 @@ private function getCacheControlDirective(ResponseInterface $response, $name)
      */
     private function createCacheKey(RequestInterface $request)
     {
+        if ('POST' === $request->getMethod()) {
+            return hash($this->config['hash_algo'], $request->getMethod().' '.$request->getUri().' '.$request->getBody());
```

Comment:
Agree on this one too. However some might consider changing the way how keys are generated a BC break. See discussion in #12. Although if I read it correctly the consensus in the en was this would not be considered a BC break.